### DISCUSSION TITLE
Can't reverify the GitHub Education

### DISCUSSION THREAD

**Discussion by Ritabear** [community/community#70221] [GitHub Education]
### Hi everyone,

Hello,

Today, I applied for the GitHub Education program to use Copilot. Unfortunately, my application was rejected, and now I'm having trouble re-verifying it. I've attempted to reapply, but I can't click the "Continue" button.

How can I go about re-verifying my GitHub Education application? Thank you for your assistance.

![image](https://github.com/community/community/assets/75626878/7b1a4c2f-29de-44eb-b1c6-940106ab8048)

### A suggestion...(optional)

Suggest having a contact email

**Comment by MisterBrash** [comment]
Hi @Ritabear - if I understand the screenshot correctly, your application was approved on your second attempt. See the green section above your highlighted red. The red section is just for historical tracking. Also, at the bottom of your screenshot there is a yellow box explaining that you are approved. It can take a long time to get past the approval process and start receiving the benefits. I am still waiting for my teacher benefits in order to use Codespaces with my classes... apparently it can take a while.
